Frameworks/AddressBook/ABMultiValue.mm

```diff
@@ -19,74 +19,108 @@
 #import <StubReturn.h>
 #import "AssertARCEnabled.h"
 
+#import "ABMultiValueInternal.h"
+
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFTypeRef ABMultiValueCopyValueAtIndex(ABMultiValueRef multiValue, CFIndex index) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (multiValue == nullptr) {
+        return nullptr;
+    }
+
+    _ABMultiValue* multi = (__bridge _ABMultiValue*)multiValue;
```

Comment:
also, confused about this __bridge cast - it doesn't appear to actually be using a bridge? you're just casting to a subclass.

---
In reply to: [75171401](https://github.com/Microsoft/WinObjC/pull/884#discussion_r75171401) [](ancestors = 75171401)